requirements.txt

```diff
@@ -1,8 +1,8 @@
-Django==2.1
+django==3.1
 grimoirelab_toolkit>=0.1.8
```

Comment:
I think we missed this. 
@jjmerchante can you please update this too?

We might remove these files soon but yeah better we keep them updated.